src/main/java/com/ibm/stocator/fs/swift/SwiftAPIDirect.java

```diff
@@ -86,6 +88,24 @@ public static SwiftInputStreamWrapper getObject(Path path, JossAccount account,
   }
 
   /**
+   *
+   */
+  public static HttpResponse getObjectMetadata(JossAccount account, String container, String object,
+                                               SwiftConnectionManager scm) {
+    HttpHead headRequest = new HttpHead(account.getAccessURL() + "/" + container + "/" + object);
+    headRequest.addHeader("X-Auth-Token", account.getAuthToken());
+    HttpClient client = scm.createHttpConnection();
+    HttpResponse metadata = null;
+    try {
+      metadata = client.execute(headRequest);
+    } catch (IOException e) {
+      LOG.error("Get object metadata operation could not be completed.");
+      e.printStackTrace();
```

Comment:
@djalova we can't just print it :) we need to use LOG and throw exception to upper layer